src/renderer/components/+helm-charts/helm-charts.tsx

```diff
@@ -76,6 +80,22 @@ class NonInjectedHelmCharts extends Component<Dependencies> {
     this.showDetails(null);
   };
 
+  private renderIcon(chart: HelmChart) {
+    const icon = chart.getIcon();
+
+    if (!icon || this.imgLoadingFailed.has(chart.getId())) {
+      return <div dangerouslySetInnerHTML={{ __html: HelmLogoPlaceholder }} />;
+    }
+
+    return (
+      <img
+        src={icon}
+        onLoad={evt => evt.currentTarget.classList.add("visible")}
+        onError={() => this.imgLoadingFailed.add(chart.getId())}
+      />
+    );
+  }
+
```

Comment:
Seems like repeatable code from `helm-chart-details.tsx`. Good chance to move it to separate component?